src/box-types.ts

```diff
@@ -0,0 +1,35 @@
+import {ReactNode, Component} from 'react'
+import * as CSS from 'csstype'
+
+type UIBoxProp = string | number | boolean | null | undefined
+export type CSSProps = CSS.StandardProperties<number | string | boolean>
+
+interface BoxPropsBase extends CSSProps {
```

Comment:
We only support a subset of CSS properties so this will result in confusing/misleading types.

Reply:
As an alternative, you'd need to export all propTypes you defined in `/enhancer` as interfaces as well and extends those in here. But they're necessary as both types and propTypes

Reply:
I used Roland's types from his branch: https://github.com/segmentio/ui-box/tree/types/types